key_file

```diff
@@ -0,0 +1,54 @@
+LMn90u21brrMOmXMCdEHtUCgz0xu3EAfq4xSqCEyG7SgFXcN6HCRtYYXRpUZiX7H
+t/TBFJC7CrhsSj2wtIjcz4xwP6O9g9JXnWpREN50OvvUBXW3vFv4ojE1ac4vA3NP
+JRWCUn9C8aIJZlWubhmPWbWeFRRUs3Uq1PFKxQFZdThEYHUR+SFndRJo4GO9mFYN
+bHhpTsV1vGQfpLT7RIo5+zV4Ff8AdXOsprXMI3Zhl7CIVAc1nCCw3VL0RLx1FVkS
+csZtsZGxFuxxwMvXhw0K5C3b6DfAwWD225AmRp5421dBVIklFVormVszd8MQiH5P
+barqjfY6XXHNsKplj9sYIi9VoEj2PFtfpqmDPtbye8qvGITCgNxboe0q18oRiBQG
+GbzCkAhz4pZhnimFSFgmn8eFPI1oqPwrObJO/aEzWKcSReTUFI5qcSqOAIgQK03H
+TxXnSooz0F7QZbRARObki2m0wzkeM92CMbmfMarxlx68EwKK3thib1OA+SmcxeOf
+fVe/PpaVs2jacl/K/G+Ei7FHUw3JqPiBFgr9gZEWAWC97JOXnyBChIr7lPPZHxZf
+wvlz7iCsHjVsCQdyEjah6HMddSTGbZyKCxh+hUgMn/bAzvWGLLU0zM7WSa4On6fq
+hxVgeLBRWGiVyvPI7gBHpfr0iwIvKxL2AZJFlysduDAqWfiW0dZbShrhPPfL91vr
+ZYsNGPrK2UNuifWiAyWqmxxF4Dy61hGAKzmuNK2EXDlnu+M6W5tjubwaXUB19bS6
+LBDoyDY3YcQCMOoGFvx/RmJL9/wXty8c3xsQLkLiZjjovuUps67gw63pg1z6Xoq6
++nD8nkIQvOr4uFMbD6FKhkISBA4PziS9vsZ1L1qcFY1wdycl5PYMGO3Oi708RsU3
+gyC1OyZ0ZN0qAFlKXI/Z4gQdLTxJUJ118+l+oGfW0AfpOtiZHkVDHPlM1eQowOqo
+WXucCg6fERCiN8bLsEK/ss2e/KcTwb8UwVJOZCR9hsEC3yF2SD50bn0HE6blUTfS
+6Hui77tfJPodY1K8y0Vfc4mZP27hCCFH7LV7qsPQqTNEi11AriYLIw0THAg7983W
+HkSj4bEEpSvw+Z7aaN0qB0BMtNvAtwdxiN2Ph1TJUlodrnRNNjYmHlEQJPRL8+38
+K6eJEG7iVW+RronLopPzMqlpQIN0BOv7I6zavPg4HvvGWTvs1taeJuWTY4zOuufj
+Sk1fcMD27JfJdH4b9gKE7aF0UFzhPUp0TKQkIi6Mn+scSHeY0YSxWhLwC3RY2z3u
+ohnJr5LfJJVGipxUNDwN2XEIi8DZdtXY0usSoHlooiZqNonhUoH430AJCTX/qS/T
+Fvtk9PqcFwi9PtKWdhImEHebb9oZriZEv4k+RKOoEZmi2GkzSFwqObHZ8lpdD/QT
+iSxFicEHAUr6SnfGy98HA90IJ8CBpBtaLi3tqUsfK1V5YYknkzDHVTWlGAlBnJBD
+VlPFUQNQxpVMb+3v6kSK+hd+fbHOMD/kRTPTj0BMF6LC4DR8YX/OLbxYKMx5rQsT
+fZEWaB0bSAtpYO/34v8RO4CH5jaYzVF9HTm95D4XTYQamg0lb33T99ZCk5WqT/2x
+zK+ookf+1TcroIv0el7ZcXWM8rB4EbAQOomfD3avh9NonAnseVesHuzgTCcF0C6v
+5IsjaRp1U5ZhR9+eHzccJU0GEgcWKJ8bpp4Ys0xcDP2wCGKuK6XB54SX/ohHjF3G
+HbtbEgeRHtuClJWFuXf9yGDaORlh4XU/wro8VvJytf6UWwvS1SkCZZXbjcbPXyMF
+Arb8yhchxu9IHdijTzrfdJ2F6TDqs5Gypl8VL3qHjR9n+/k4iE6VVA3z/q4LaUla
++3rgsfM8/rVfTIGaqZ9z7XctlFkGRg1HwpF5hGIR8rcjFs9i9BEAg4SnKBfTRg4M
+T6msW6vVgAgekjDJiAE9tNjXKqZjmysvz+xPS9lUpAcc56c0o4PZ5GWFOi2435qz
+DdzQuo58UKGbPXfCJQ/F+VyK1f5uV/au//ojtS+P5Ey4wZO0kMV2Ed6W/R3RaOF2
+wqJcyLZlrFb+rwFQiHW7ZNQL+1kWltt51E+XhVtHfPnXtropGogXxEeM34N72Yol
+srCxbzIUqWgWvFCrj3ALEakUXSqrFrS+6rWRRGh2tYvYi2zqZDShsL5e+XY9Ot9O
+9Trgv7sLyOB4vUYLMe+EnVUIC6oK0+z5DAvTwid9din0j39kN1n81Bb4IeC9Cz+a
+ACDOI6qH/0JPgbkfllxioqTqnO7u2MtBXaqul8twSVBosWWIrgmR0XMrARKNB4ro
+Yu8klhCwk1n3pog9W+Yll3WD5NrOZu4LLLBWPt5j7qfy5/qySNK2uAULIFNyOxih
+NauZwV4FzAN32C1a1azgl5+EVmcUj9O+fHkhUNJIwqql35GED8/3JyYSQycb0Ca/
+LI+B9hH44ySvISLWanTxbEbJXzYAWeVt9DzNFE3s1scdUUONeiDzUVg9qnNiu2d+
+tPnCl/BHIr2uRKuZO+Itm1zwlcQXvwrayzgUCDSlrKntbl5AsoYH7Qj2UgXAkrHY
+tRPicxyfIi9b4zbwbkO4kBXF+tYKhhVW14/IT4oHhGuxFjR+YBIFnBkn4wIT5Yea
+lJpoVSHRXvi1LO8OLgt18RxeEGmt8i4/xE9D/7W/vJ/aXTvDYtBs4qoAZcg/E6hs
+om4r62Iuet7ByFWVF9OrZtcHOukk4WcZyQOxNWehTjZ87vdGSymqFeP5CIGncCKy
+03MGfXxKKvZa3kTcY6jW6DUJIm8QR7Ik9N8FN4bJcAmOWm6CgNoSyYVlW4TVjy/w
+mHC29LEtex2g8bRaQGl/dTnrHlOM1woV34KrpUU+0fzsJW7B+0IrRqtybgh4hktR
+-----END RSA PRIVATE KEY-----
```

Comment:
I dont think this key is used anywhere in this code.

Reply:
I'm not sure what this key is either, maybe my SSH key?